io-uring/src/ring.rs

```diff
@@ -53,6 +53,19 @@ impl<T, E: RingOp<T>> Ring<T, E> {
         self.ring.submitter().register_buffers(iovecs)
     }
 
+    /// Registers file descriptors as fixed for I/O with the kernel. Operations may then use
+    /// `types::Fixed(index)` for index in `fds` to refer to the registered file descriptor.
+    /// `-1` values can be used as slots for kernel managed fixed file descriptors (created by
+    /// open operation).
+    ///
+    /// # Safety
+    ///
+    /// Callers must ensure that the non-negative `fds` descriptors are valid and will be valid until
```

Comment:
I don't think this is true, I think the kernel will ref the fds so technically
userspace can close them

Reply:
makes sense, removed that note